tests/Functional/LegacyAPITest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Doctrine\DBAL\Tests\Functional;
+
+use Doctrine\DBAL\FetchMode;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+
+use function array_change_key_case;
+use function array_map;
+
+use const CASE_LOWER;
+
+class LegacyAPITest extends FunctionalTestCase
+{
+    /** @var bool */
+    private static $generated = false;
```

Comment:
It's better to use `dropAndCreateTable()` and make sure that the setup always produces the same sate just by that.

Reply:
Ah ok, i was just replicating what `DataAccessTest` does.